test/flink_test.go

```diff
@@ -120,3 +152,214 @@ func (s *CLITestSuite) TestFlink_Autocomplete() {
 		s.runIntegrationTest(test)
 	}
 }
+
+func (s *CLITestSuite) TestFlinkShell() {
+	tests := []FlinkShellTest{
```

Comment:
nit: wdyt of listing tests from folder to exclude one step when adding test?


```
    files, err := os.ReadDir('input/flink/shell')
    if err != nil {
        fmt.Println("Error reading directory:", err)
        return
    }

    for _, file := range files {
         testName := file.Name()
         s.runFlinkShellTest({{inputFile: testName, goldenFile: testName + ".golden" }})
    }
```

Reply:
Or better yet, we could just use a `[]string` instead of a file for the input? A good test will have a short script of statements so this should be fine.

Reply:
yeah sounds good, I wanted to make use of the input folder 😄 Using a string array now instead!